tests/test_persistedqueue.py

```diff
@@ -0,0 +1,66 @@
+import aioredis
+
+from acapy_plugin_pickup.protocol.delivery import PersistedQueue
+from aries_cloudagent.transport.outbound.message import OutboundMessage
+import pytest
+
+
+@pytest.fixture
+def target():
+    yield {
+        "did": "some_did",
+        "endpoint": "some_endpoint",
+        "label": "some_label",
+        "recipient_keys": "recipient_keys",
+        "routing_keys": "routing_keys",
+        "sender_key": "some_sender_key",
+    }
+
+
+@pytest.fixture
+def msg(target):
+    yield OutboundMessage(
+        connection_id="conn_id",
+        target=target,
+        reply_from_verkey="reply_from_verkey",
+        payload="payload",
+    )
+
+
+@pytest.mark.asyncio
+async def test_persistedqueue(msg):
+    """
+    PersistedQueue Test.
+
+    Unit test for the PersistedQueue class in
+    """
+    PQ = PersistedQueue(redis=await aioredis.from_url("redis://localhost/1"))
```

Comment:
This is one of our problems. This test should be in the integration tests, not in the unit tests. Tests that depend on external services are by definition integration tests or "higher" in the testing hierarchy (unit < int < system < ....)